paparazzi-annotations/src/main/java/app/cash/paparazzi/annotations/PaparazziPreviewData.kt

```diff
@@ -0,0 +1,31 @@
+package app.cash.paparazzi.annotations
+
+import androidx.compose.runtime.Composable
+
+/**
+ * Represents composables annotated with @Paparazzi annotation
+ *
+ * Default - Represents a composable with no parameters
+ * Empty - Represents a configuration with zero annotated composables
+ * Error - Represents an error state with a message if the composable is misconfgured (ex. private Composable function)
+ */
+public sealed interface PaparazziPreviewData {
+
+  public data class Default(
+    val snapshotName: String,
+    val composable: @Composable () -> Unit
+  ) : PaparazziPreviewData {
+    override fun toString(): String = snapshotName
+  }
+
+  public data object Empty : PaparazziPreviewData {
+    override fun toString(): String = "Empty"
```

Comment:
`data object` automatically generates `toString()` to return the object name.

<img width="70%" alt="image" src="https://github.com/user-attachments/assets/44d5581a-af78-48d8-ab87-4fb159a77963">

```suggestion

```